### ISSUE TITLE
Holistic style design

### ISSUE THREAD

**Issue by ZeLonewolf** [ZeLonewolf/openstreetmap-americana#7]
A holistic design is desired for the project which would provide contributors with guidance in implementing features, which includes but is not limited to:
* Which features to emphasize, which features to de-emphasize, and which features to exclude entirely.
* Stylistic purpose at each zoom level (or cluster of zoom levels)
* Color assignments

It is desired that a holistic design be inspired by American aesthetics.

**Comment by Mashin6** [comment]
Do you mean something like a digital map with open source style or some kind of traditional paper map? One has to be careful about the copyright issues.

**Comment by ZeLonewolf** [comment]
I mean that our map should look and feel familiar to Americans.  So, we would want design features that are common across a wide variety of North American makes (for example, highway shields, text labels alongside roads rather than squeezed in the fill).  We would want to emphasize and de-emphasize features that Americans would expect to see in a map at a certain zoom.  So, we should not copy a particular map, but we should consider which design features tend to appear across a wide variety of them.

**Comment by Mashin6** [comment]
Gotcha. 
Maybe we could post here more links to printed or online maps that we could use to get a sense of what a typical US map look like.

Also... related to #25. We should decide what the general purpose of the map would be. There are different types of maps that serve particular purpose, which then dictates how and which features are shown. There is a difference between a road map for car traveling, hiking map, public transport map, city plan,...
Because the highway shields are so prominent in this project, I assume you want to more gear it towards a road map for people that are in their cars looking for a business or a place to go..

**Comment by ZeLonewolf** [comment]
> We should decide what the general purpose of the map would be. There are different types of maps that serve particular purpose, which then dictates how and which features are shown. There is a difference between a road map for car traveling, hiking map, public transport map, city plan

Agreed, however, we're not as tied to a singular purpose in the way that a paper map is, because paper maps are stuck at a single zoom level.  Rather than say "this map is for purpose X", I'd rather say "these features are emphasized and these features are de-emphasized", and also consider what makes sense based on zoom level.

Here are some initial thoughts that I had on map purpose:

**"If you were travelling to an unfamiliar place, what would you want to see on a map?"**

Thus, I think it is extremely important that a map strikes the right balance of showing useful information while minimizing clutter, and providing "breadcrumbs" to indicate where more information is available by zooming in.

The real challenges will be in the middle zooms, i.e. in the raster 12-16 (vector 11-15) range.  That's where styles like openstreetmap-carto have the biggest issue with too-high information density overwhelming the map.  We should think really carefully about what information do we want to convey at each zoom while still leaving "breadcrumbs" to tell map users that there's more information available by zooming in.

Also, as a general philosophy, and consistent with making an "American-style" map, we should de-emphasize/simplify/unify tagging schemes that doesn't fit the US very well.

Given the American reliance on the car, and the fact that road networks are the most important form of transportation in the US, roads, highway shields, and route networks are definitely something I think we should emphasize.  However, roads are not in competition with hiking areas - both should co-exist since they're in different places.  I would like to be able to emphasize long-distance hiking trails, such as the Appalachian Trail, Long Trail, North South Trail, New England Trail, Pacific Crest Trail, and so forth.  Those exist in places where there aren't highways and are interesting American features.

I think we want to not render OSM land cover at the mid-zooms, because the coverage is so poor in the US, for example:
![image](https://user-images.githubusercontent.com/3254090/125146581-869cd800-e0f4-11eb-80b1-3ee4a0c2cd98.png)

If we're not rendering or de-emphasizing land cover, it may make sense to simplify or even unify the rendering style for parks, nature reserves, and protected areas, as the distinction between them is quite blurry in the US.

At the highest (closest) zoom levels, we should try to render basically everything, because nothing will be in competition with each other.  If you're close enough that it makes sense to show a fire hydrant or a manhole cover, there's no reason not to render it.

I would also like to see us do external integrations, especially with wikidata.  For example, I would 100% be in favor of showing corporate logos on stores at close zoom, by pulling them out of `brand:wikidata` tags and doing a lookup into wikidata.

**Comment by Mashin6** [comment]
I started a draft here: https://wiki.openstreetmap.org/wiki/United_States/Map_style/Holistic_design
